blueocean-personalization/src/main/js/jenkins-js-extension.yaml

```diff
@@ -2,8 +2,6 @@
 # NB: "component" currently maps to modules, not "symbols" so make sure to "export default"
 # WARNING: If you change this you'll have to change io.jenkins.blueocean.jsextensions.JenkinsJSExtensionsTest as well :(
 extensions:
-  - component: redux/FavoritesStore
```

Comment:
This is the problem causing `io.jenkins.blueocean.config.JenkinsJSExtensionsTest` to fail. We should just delete that test, it's useless.

Reply:
yeah I agree. It made sense early on but not relevant now.